### PR TITLE
Wip/hughsie/sign

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

File: lvfs/firmware/routes.py [hunk 1]
```diff
@@ -179,6 +179,29 @@ def route_nuke(firmware_id):
     flash('Firmware nuked', 'info')
     return redirect(url_for('firmware.route_firmware'))
 
+@bp_firmware.route('/<int:firmware_id>/resign')
+@login_required
+@admin_login_required
+def route_resign(firmware_id):
+    """ Delete a firmware entry and also delete the file from disk """
```
Comment (superm1) [inline lvfs/firmware/routes.py]: need to update comment

File: lvfs/firmware/routes.py [hunk 2]
```diff
@@ -179,6 +179,29 @@ def route_nuke(firmware_id):
...
+
+    # check firmware exists in database
+    fw = db.session.query(Firmware).filter(Firmware.firmware_id == firmware_id).first()
+    if not fw:
+        flash('No firmware {} exists'.format(firmware_id), 'danger')
+        return redirect(url_for('firmware.route_firmware'))
+
+    # firmware is not deleted yet
```
Comment (superm1) [inline lvfs/firmware/routes.py]: eh?

Comment (hughsie) [reply lvfs/firmware/routes.py]: Fixed, thanks.

File: lvfs/firmware/routes.py (hunk 1 above)
Comment (hughsie) [reply lvfs/firmware/routes.py]: Fixed, thanks.

**Pull request merged**